### ISSUE TITLE
Clicking error message in console does not take editor to appropriate line

### ISSUE THREAD

**Issue by BlacksheepVMF-214** [spyder-ide/spyder#15545]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
While running a Python program, an error message is created in the console. Clicking the error message should take me to the appropriate line in the Python code (and it did so in earlier versions of Spyder), but it does not now. The mouse pointer changes to a finger when hovering over the error message, but clicking has no effect.

### What steps reproduce the problem?
1. Run a Python program that has a coding error. An error message is generated in the console; this error message has an associated line number from the Python program.
2. Click the generated error message in the console. Nothing happens.

### What is the expected output? What do you see instead?
I expect that clicking the error message in the console would move the current line in the editor to the corresponding line that contains the error.
What I see instead is that nothing happens.

### Paste Traceback/Error Below (if applicable)

```python-traceback

N/A. There is no Python error generated by the problem.

```

## Versions

* Spyder version: 5.0.0 None
* Python version: 3.8.8 64-bit
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10
* Spyder version: 
* Python version: 
* Qt version: 
* PyQt version: 
* Operating System name/version: 

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  22.3.0 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                :  3.5.3 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  20.0.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  None (NOK)
numpy >=1.7                   :  None (NOK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  None (NOK)

```

**Comment by juanis2112** [comment]
Hi @BlacksheepVMF-214. Thanks for reporting this. Please upgrade your Spyder to 5.0.1 and let us know if the issue persists. You can download it from out webpage https://www.spyder-ide.org/#section-download

**Comment by BlacksheepVMF-214** [comment]
My apologies for the delay in responding; things got in the way (work, life, ...).

After several false starts, I finally got Spyder 5.0.2 correctly installed; I am no longer having this issue.

This also solved an additional issue: The text of menu options was overlapping their associated icons.

Thank you, and have a great day.

**Comment by ccordoba12** [comment]
Great! Thanks for letting us know about it.

**Comment by pyLP7** [comment]
Hi @ccordoba12, @BlacksheepVMF-214,

I am actually facing a similar issue here. I got the latest Spyder version available on Anaconda, i.e. Spyder 5.1.5. 

The mouse pointer is not even changing into a finger when I move it over the error message and even if I click (nor double click) nothing happens.
![grafik](https://user-images.githubusercontent.com/73650883/172885776-49105f0b-4f37-4c47-b863-57ec54665148.png)

Any idea why? Thanks!

**Comment by ccordoba12** [comment]
@pyLP7, this should be fixed in the latest version available in Anaconda (5.1.5). You're probably using an older version, so please update.

**Comment by pyLP7** [comment]
The second one worked fine for me. Thank you very much @ccordoba12!

**Comment by luca-morino** [comment]
Hi! Had the same problem as you. If you want to stay in Anaconda then my workaround was to downgrade the IPython console to 7.31.1

**Comment by DVPombo** [comment]
@ccordoba12 I have tried the steps you mention without luck. Any other suggestions?

**Comment by DVPombo** [comment]
@ccordoba12 After a few tries your steps worked. I needed to install 
pip install spyder-kernels==2.3.*
in order to make it work.
However, the cursor does not turn into a hand anymore. But at least it takes me to the error line.

**Comment by ccordoba12** [comment]
> However, the cursor does not turn into a hand anymore. But at least it takes me to the error line.

Yeah, I introduced that small regression last year but I'll try to fix it in our next version.

**Comment by marianux** [comment]
Hi, I am facing this same issue. Any idea or workaround ? Thanks in advance!


* Spyder version: 5.4.2  (pip)
* Python version: 3.10.6 64-bit
* Qt version: 5.15.2
* PyQt5 version: 5.15.9
* Operating System: Linux 5.15.0-67-generic

# Mandatory:
atomicwrites >=1.2.0          :  1.4.1 (OK)
chardet >=2.0.0               :  5.1.0 (OK)
cloudpickle >=0.5.0           :  2.2.1 (OK)
cookiecutter >=1.6.0          :  2.1.1 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.31.1;<9.0.0       :  8.11.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.2 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.17.3 (OK)
keyring >=17.0.0              :  23.13.1 (OK)
nbconvert >=4.0               :  7.2.9 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.4 (OK)
pygments >=2.0                :  2.14.0 (OK)
pylint >=2.5.0;<3.0           :  2.16.3 (OK)
pylint_venv >=2.1.1           :  3.0.1 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.7.1;<1.8.0          :  1.7.1 (OK)
pylsp_black >=1.2.0           :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.3 (OK)
qstylizer >=0.2.2             :  0.2.2 (OK)
qtawesome >=1.2.1             :  1.2.2 (OK)
qtconsole >=5.4.0;<5.5.0      :  5.4.0 (OK)
qtpy >=2.1.0                  :  2.3.0 (OK)
rtree >=0.9.7                 :  1.0.1 (OK)
setuptools >=49.6.0           :  67.4.0 (OK)
sphinx >=0.6.6                :  6.1.3 (OK)
spyder_kernels >=2.4.2;<2.5.0 :  2.4.2 (OK)
textdistance >=4.2.0          :  4.5.0 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.3.1 (OK)
xdg >=0.26                    :  0.28 (OK)
zmq >=22.1.0                  :  25.0.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=3.0.0            :  3.7.0 (OK)
numpy >=1.7                   :  1.24.2 (OK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  1.9.3 (OK)
sympy >=0.7.3                 :  1.11.1 (OK)

**Comment by ccordoba12** [comment]
@marianux, this problem resurfaced again with IPython 8. We'll try to fix it in our next version (5.4.3), to be released at the end of the month.

**Comment by Miguel-LlamasLanza** [comment]
Can we downgrade ipython to 7.x, as a workaround until it is solved in 5.4.3? Or will this result in any issue?

**Comment by AdrienMau** [comment]
Same error here with a brand new computer on Windows 11 and a new Anaconda3 installation, with Spyder 5.4.1, Ipython 8.10.0 and Python 310.9.
Cannot click the error, cursor does not change.

**Comment by ccordoba12** [comment]
@AdrienMau, this problem is fixed in our latest version (5.4.3), released last month. So, please update to it.

**Comment by jna00** [comment]
It is possible this has again resurfaced. I installed spyder yesterday and the jump to the error line from the IPyhon interprterer doesn't seem to worlk. 

* Spyder version: 5.5.3  (conda)
* Python version: 3.12.2 64-bit
* Qt version: 5.15.8
* PyQt5 version: 5.15.9
* Operating System: Windows-10-10.0.19045-SP0
IPython 8.22.2 -- An enhanced Interactive Python.
